docs/features.md

```diff
@@ -327,11 +351,13 @@ You can use both sync and async functions for middlewares!
 ```python
 @app.before_request("/")
 async def hello_before_request(request):
```

Comment:
```suggestion
async def hello_before_request(request: Request):
```

Reply:
Ooh , playing typed vs untyped Python, are we? 🤣 . I will update it.

Reply:
Ahah my opinion on the subject is that everything should be typed because it allows to catch more errors and provides better hints when coding ^^